src/main/java/com/evolutiongaming/prometheus/cassandra/TimerMetricFamilyBuilder.java

```diff
@@ -49,35 +49,27 @@ private void addQuantileMetric(List<String> labelValues, String quantile, double
     List<String> quantileLabelValues = new ArrayList<>(labelValues);
     quantileLabelValues.add(quantile);
     quantileSamples.add(new Collector.MetricFamilySamples.Sample(
-        name, quantileLabelNames, quantileLabelValues, value
-    ));
+        name, quantileLabelNames, quantileLabelValues, value));
   }
 
   private void addCountMetric(List<String> labelValues, double value) {
     countSamples.add(new Collector.MetricFamilySamples.Sample(
-        name + "_count", labelNames, labelValues, value
-    ));
-  }
-
-  private void addMeanMetric(List<String> labelValues, double value) {
-    meanSamples.add(new Collector.MetricFamilySamples.Sample(
-        name + "_mean", labelNames, labelValues, value
```

Comment:
What's the point of changing the formatting here? I'd rather change only the lines which need to be changed.

Reply:
Yeah, sorry. My editor is still unaccustomed to editing java, and currently, I just constantly fight with autoformatting. Reverted.